validator/sawtooth_validator/journal/publisher.py

```diff
@@ -35,6 +37,30 @@
 LOGGER = logging.getLogger(__name__)
 
 
+class TransactionSender(object):
```

Comment:
TransactionSender seems like a bit of a misnomer. Transactions don't get sent, they get packaged into batches which get sent via the BatchSender. Maybe BatchBuilder and then BatchSender would be more natural?